nomenclature/__init__.py

```diff
@@ -49,6 +49,10 @@ def _copy_dict(dct, description):
 
 # explode <Fuels> tags to full lists
 fuel_types = _variables.pop('<Fuel>')
+# explode <industry> tags to full lists
+industry_types = _variables.pop('<industry>')
+# explode <product> tags to full lists
+product_types = _variables.pop('<product>')
```

Comment:
We use here for the different tags (`<Fuel>`, `<industry>`, `<product>`) only once a capital letter.

Reply:
Got it! Thx.